packages/block-editor/src/store/selectors.js

```diff
@@ -1780,19 +1783,43 @@ export function canLockBlockType( state, nameOrType ) {
 	return !! state.settings?.canLockBlocks;
 }
 
+/**
+ * Return all insert usage stats.
+ *
+ * This is only exported since registry selectors need to be exported. It's marked
+ * as unstable so that it's not considered part of the public API.
+ *
+ * @return {Object<string,Object>} An object with an `id` key representing the type
+ *                                 of block and an object value that contains
+ *                                 block insertion statistics.
+ */
+export const __unstableGetInsertUsage = createRegistrySelector(
```

Comment:
I should make the __unstable selectors private instead (the PR originally predated the concept of private selectors)

Reply:
It's not a particularly complicated function. Instead of having it private / unstable could any callers that need it just call `select( preferencesStore ).get` directly?

Alternatively it can stay as a non-exported function but have it so that callers pass in `select` as an argument.

Same for `__unstableGetInsertUsageForBlock`.